frontend/ui/lib/widgets/trip_list_widget.dart

```diff
@@ -35,8 +36,33 @@ class TripListWidget extends StatelessWidget {
                 Navigator.pushNamed(
                     context, Router.createTripViewRoute(trip.id));
               },
+              trailing: Tooltip(
+                  message: 'Click here to delete trip.',
```

Comment:
Make the `IconButton` the top-level widget here and make the tooltip a property of that button.